packages/polling-controller/src/PollingController.ts

```diff
@@ -6,6 +6,11 @@ import { v4 as random } from 'uuid';
 // See TS2545
 type Constructor = new (...args: any[]) => object;
 
+// get unique key for outer poll token
```

Comment:
Instead of just adding some comments above the function, what do you think about using JSDoc, so that the docs appear when hovering over the function?